agenta-backend/agenta_backend/models/api/user_models.py

```diff
@@ -12,7 +12,7 @@ class User(TimestampModel):
     uid: str
     username: str
     email: str  # switch to EmailStr when langchain support pydantic>=2.1
```

Comment:
I don\t understand the comment. what does this have to do with langchain

Reply:
I met the comment here, I think what @aybruhm meant was to declare the type as EmailStr from pydantic but dependencies between langchain and pydantic clash.